JitsiConferenceEventManager.js

```diff
@@ -380,10 +380,11 @@ JitsiConferenceEventManager.prototype.setupChatRoomListeners = function() {
         XMPPEvents.REACTION_RECEIVED,
 
         (jid, reactionList, messageId) => {
+            participantId = Strophe.getResourceFromJid(jid);
```

Comment:
Please make sure you add const in front of this so it's defined, as it's a new variable without context